lisa/runner.py

```diff
@@ -220,7 +220,7 @@ async def start(self) -> None:
 
             self._start_loop()
         except Exception as identifier:
-            self._log.info(f"canceling runner due to exception: {identifier}")
+            self._log.exception(f"canceling runner due to exception: {identifier}")
```

Comment:
I think the exception message should be printed by the `log.exception` already. It doesn't need to be included in the custom message again.

Reply:
It is with the following format:
```
canceling runner due to exception: {identifier}
Stacktrace
{identifier}
```

I think we can leave it here since it provides more context to the message `canceling runner due to exception`.

Reply:
@somil55 any update?